### DISCUSSION TITLE
No longer synching

### DISCUSSION THREAD

**Discussion by Hermodion** [derekantrican/GAS-ICS-Sync#330] [Q&A]
I had this installed and running, but it stopped at some point.  I uninstalled and installed without making any changes and it throws the following error:

Mar 30, 2023, 1:15:11 PM	Error	DNS error: http://icsUrl1
    at callWithBackoff(Helpers:939:9)
    at fetchSourceCalendars(Helpers:80:5)
    at startSync(Code:159:21)

I verified the URL is the same as before when it was working.  Not sure how to resolve.
Thanks!

**Comment by VNRARA** [comment]
Script has been ffing up my agenda too. Half of the events don't show.
